app/src/pages/settings/CreateSystemAPIKeyDialog.tsx

```diff
@@ -123,6 +155,26 @@ export function CreateSystemAPIKeyDialog(props: {
               />
             )}
           />
+          <Controller
+            name="expiresAt"
+            control={control}
+            render={({
+              field: { name, onChange, onBlur, value },
+              fieldState: { invalid, error },
+            }) => (
+              <TextField
+                label="Expires At"
+                type="datetime-local"
+                name={name}
+                description={`The date at which the key will expire. Optional`}
```

Comment:
if not set then there is just no expiry on the key, correct?

Reply:
Yeah